tests/api/test_swagger_ui.py

```diff
@@ -1,4 +1,15 @@
+from connexion.spec import Specification
+
+
 def test_simple(swagger_ui_app):
     app_client = swagger_ui_app.test_client()
     response = app_client.get("/v1.0/spec.json")
     assert response.status_code == 200
+    Specification.from_dict(response.json())
```

Comment:
Added a comment, at first tried to explicitly test, however this would add a lot of logic to test which spec version it is so we can load the correct JSON Schema to validate using the spec validator.